v20.2/backup.md

```diff
@@ -35,9 +35,14 @@ To view the contents of an enterprise backup created with the `BACKUP` statement
 
 ## Required privileges
 
-- Only members of the `admin` role can run `BACKUP`. By default, the `root` user belongs to the `admin` role.
+- Full cluster backups can only be run by members of the `admin` role. By default, the `root` user belongs to the `admin` role.
```

Comment:
~Could "full cluster backups" link to #backup-types or #full-backups ?  Could "admin role" link to some docs on the admin role?  Asking because I am the user who has no idea what our roles etc. are and would have to search every time